.github/workflows/kubernetes.yaml

```diff
@@ -200,6 +210,27 @@ jobs:
             ${{ inputs.registryHostname }}/${{ inputs.registryOrg }}/${{ github.event.deployment.payload.name }}:latest
             ${{ inputs.registryHostname }}/${{ inputs.registryOrg }}/${{ github.event.deployment.payload.name }}:${{ steps.vars.outputs.version }}
             ${{ inputs.registryHostname }}/${{ inputs.registryOrg }}/${{ github.event.deployment.payload.name }}:${{ github.sha }}
+      - name: Build and push primary image to GitHub
```

Comment:
Can we adjust the title using ${{ matrix.containerfile_targets }} ?

Reply:
Done

Reply:
```Build and push ${{ matrix.containerfile_targets }} image to GitHub```